vicare/climate.py

```diff
@@ -145,12 +145,14 @@ def update(self):
             else:
                 self._current_temperature = None
             self._current_program = self._api.getActiveProgram()
-
+            try:
```

Comment:
We will need the try catch on every property, since the execution will stop on the first exception.

Reply:
I guess so, I've just made this quick fix to make it work :)